src/pcapi/core/offers/exceptions.py

```diff
@@ -44,6 +44,38 @@ def __init__(self):
         super().__init__("Nous n'avons pas réceptionné l'image, merci d'essayer à nouveau.")
 
 
+class SubcategoryNotEligibleForEducationalOffer(ClientError):
+    def __init__(self):
+        super().__init__(
+            "offer",
+            "Cette catégorie d'offre n'est pas éligible aux offres éducationnelles",
+        )
+
+
+class OfferCannotBeDuoAndEducational(ClientError):
+    def __init__(self):
+        super().__init__(
+            "offer",
+            "Une offre ne peut être à la fois 'duo' et 'éducationnelle'.",
+        )
+
+
+class UnknownOfferSubCategory(ClientError):
+    def __init__(self):
+        super().__init__(
+            "subcategory",
+            "La sous-catégorie de cette offre est inconnue",
+        )
+
+
+class SubCategoryIsNotSelectable(ClientError):
```

Comment:
```suggestion
class SubCategoryIsInactive(ClientError):
```
Ce ne serait pas plus clair / exact ?

Reply:
Si complètement!
Tiens d'ailleurs, comme elle n'est pas visible au front, je renverrai bien une 500 (cf le talk de Quentin hier => si cette erreur est levée, c'est soit que le front a un problème et a permis à un acteur de sélectionner cette sous-catégorie, soit qu'un petit malin a essayé de modifier la requête :))

Reply:
On peut en effet, mais ce n'était pas la politique de l'API jusqu'à maintenant.
Voir avec les pros si ça leur va.